testing/integration/package_version_test.go

```diff
@@ -102,23 +103,40 @@ func TestComponentBuildHashInDiagnostics(t *testing.T) {
 			return false
 		}
 
+		if client.State(status.State) != client.Healthy {
+			stateBuff.WriteString(fmt.Sprintf(
+				"agent isn't healthy: %s-%s",
```

Comment:
Maybe `agent is unhealthy` would be clearer?

Reply:
there are more states than just healthy and unhealthy, that's the reason I chose this phrasing.